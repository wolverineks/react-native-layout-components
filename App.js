// @flow

import React from 'react'
import { StyleSheet, Text, View } from 'react-native'

export type Props = {||}

export default class App extends React.Component<Props> {
  render () {
    return (
      <View style={styles.container}>
        <Text>Open up App.js to start working on your app!</Text>
      </View>
    )
  }
}

const THEME = {
  COLORS: {
    WHITE: 'white'
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: THEME.COLORS.WHITE,
    alignItems: 'center',
    justifyContent: 'center'
  }
})
