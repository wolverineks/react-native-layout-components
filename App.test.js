// @flow
/* globals jest describe it expect */

import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'

import App from './App'

describe('App', () => {
  it('should render', () => {
    const renderer = new ShallowRenderer()
    const props = {}
    const actual = renderer.render(<App {...props} />)

    expect(actual).toMatchSnapshot()
  })
})