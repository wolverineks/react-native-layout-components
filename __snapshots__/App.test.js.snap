// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Open up App.js to start working on your app!
  </Text>
</Component>
`;
